src/cloud-api-adaptor/ibmcloud/image/import.sh

```diff
@@ -77,7 +79,7 @@ if [ -f ${image_file} ]; then
 else
     # Download image
     echo "Downloading file from image ${image_file}"
-    file=$($script_dir/../../podvm/hack/download-image.sh ${image_file} $script_dir) || error "Unable to download ${image_file}"
+    file=$($script_dir/../../podvm/hack/download-image.sh ${image_file} . --platform "${platform}") || error "Unable to download ${image_file}"
```

Comment:
What was the reason for changing the download directory from `$script_dir` to `.`? I'm not sure if this will impact some existing scripts, so would like to understand it more.

Reply:
Oh right, I forgot I changed that. It's a minor improvement. Previously, the script only worked if you ran it in the script dir (i.e., `./import.sh ...`), because the later upload used the current directory to find the image file. With this change, you can run the script from whatever directory you want (e.g., cd ~/work; path/to/import.sh ...`).